client/service/custom_server_manager.go

```diff
@@ -282,3 +285,92 @@ func (m *CustomServerManager) newSelectorOutbound(outboundManager adapter.Outbou
 
 	return nil
 }
+
+func insecureHttpClient() http.Client {
+	transport := http.Transport{
+		TLSClientConfig: &tls.Config{
+			InsecureSkipVerify: true,
```

Comment:
This seems tricky in terms of the potential to for adversaries to MITM traffic...

Can we just issue trusted certs?

Reply:
@myleshorton hm. and how would we do that? for that we need to be the signing authority, right? so we can't just generate self-signed certificates on lantern-server-manager startup.
Could you elaborate what did you have in mind?

Reply:
Can't we use let's encrypt and just trust their root CAs?

Reply:
@myleshorton how can we use Let's Encrypt if the user doesn't have a domain name? (which I think most people won't)

Reply:
Ah of course got it. I wonder if we can make the public key part of the config somehow? When we create these servers, do any of them have generic domains we can use? Like doesn't AWS assign some super generic subdomain to each ec2 instance, for example?

Reply:
> I think the only way to avoid having un-trusted TLS is - on start of Lantern Server Manager we contact Lantern API that will sign the certificate.

Makes sense. What if we did it in code on the server manager side? As in:

1. Have the server manager generate a new cert on the first run
2. Call our API to get it signed by our own trusted root CA or whatever
3. Have the client trust that root CA

Reply:
Ha just realized that's what you're saying....yes I think that sounds right!